core/src/main/scala/org/apache/spark/sql/delta/commands/ShowTableColumnsCommand.scala

```diff
@@ -0,0 +1,150 @@
+/*
+ * Copyright (2021) The Delta Lake Project Authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.spark.sql.delta.commands
+
+import org.apache.hadoop.fs.Path
+import org.apache.spark.sql.{Row, SparkSession}
+import org.apache.spark.sql.catalyst.{CatalystTypeConverters, ScalaReflection, TableIdentifier}
+import org.apache.spark.sql.catalyst.analysis.{NoSuchDatabaseException, NoSuchTableException}
+import org.apache.spark.sql.catalyst.catalog.CatalogTable
+import org.apache.spark.sql.catalyst.encoders.ExpressionEncoder
+import org.apache.spark.sql.catalyst.expressions.Attribute
+import org.apache.spark.sql.delta.commands.TableColumns.toRow
+import org.apache.spark.sql.delta.{DeltaErrors, DeltaLog, DeltaTableIdentifier}
+import org.apache.spark.sql.execution.command.LeafRunnableCommand
+import org.apache.spark.sql.types.StructType
+
+/**
+ * The column format of the result returned by the `SHOW COLUMNS` command.
+ */
+case class TableColumns(columnName: String)
+
+object TableColumns {
+  val schema: StructType =
+    ScalaReflection.schemaFor[TableColumns].dataType.asInstanceOf[StructType]
+
+  private lazy val converter: TableColumns => Row = {
+    val toInternalRow = CatalystTypeConverters.createToCatalystConverter(schema)
+    val toExternalRow = CatalystTypeConverters.createToScalaConverter(schema)
+    toInternalRow.andThen(toExternalRow).asInstanceOf[TableColumns => Row]
+  }
```

Comment:
These are copied from `Describe Details` command. I was wondering it might be helpful later when adding more informations to this command. Will simplify it

Reply:
Fixed.